app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -135,16 +135,17 @@ protected function isAdminAndUseJsInAdmin($checkoutType)
      */
     public function buildCartData($orderCreationResponse, $checkoutType = null)
     {
-        $authCapture = $checkoutType === 'admin' ? true : Mage::getStoreConfigFlag('payment/boltpay/auto_capture');
-
         //////////////////////////////////////////////////////////////////////////
         // Generate JSON cart and hints objects for the javascript returned below.
         //////////////////////////////////////////////////////////////////////////
         $cartData = array(
-            'authcapture' => $authCapture,
             'orderToken' => ($orderCreationResponse) ? $orderCreationResponse->token: '',
         );
 
+        if($checkoutType !== self::CHECKOUT_TYPE_ADMIN) {
```

Comment:
@johnmccombs1, @matt-thomason 

If we are moving auto-capture setting to the Bolt Merchant Dashboard, then this should not be included in the Bolt order creation data, correct?

Also, you can remove the configuration from the Magento admin in configuration.

Reply:
Agreed here @johnmccombs1